opl/pass_or_fail.py

```diff
@@ -114,12 +114,18 @@ def main():
     if args.history_type == "csv":
         history = opl.investigator.csv_loader.load(args.history_file, args.sets)
     elif args.history_type == "elasticsearch":
+        if hasattr(args, "es_server_verify"):
```

Comment:
I assume you need this, or am I missing something?
```suggestion
        if hasattr(args, "es_server_verify") and args.es_server_verify == False:
```